core/tests/protractor/embedding.js

```diff
@@ -22,11 +22,89 @@ var users = require('../protractor_utils/users.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var editor = require('../protractor_utils/editor.js');
 var ExplorationPlayerPage =
-  require('../protractor_utils/ExplorationPlayerPage.js');
+  require('../protractor_utils/ExplorationPlayerPage.js')
+var workflow = require('../protractor_utils/workflow.js');
 
 describe('Embedding', function() {
   var adminPage = null;
   var explorationPlayerPage = null;
+  var explorationId = null;
+  var createCountingExploration = function (){
+    // Intro
+    editor.setStateName('Intro');
+    editor.setContent(forms.toRichText(
```

Comment:
(Optional) Just so you know, it's absolutely fine to make these things shorter and use only as many responses as needed (so, no need to input the extraneous ones). That might speed things up. The point of the protractor test is to test the embedding _functionality_, so the specific content doesn't matter too much, as long as the necessary elements for the test are there.

Reply:
ok, i will add short meaning full sentences.

Reply:
done.